tests/unit/worker/worker_test.py

```diff
@@ -335,3 +338,64 @@ def test_get_jobs():
                                                                 max_jobs_to_activate=task.max_jobs_to_activate,
                                                                 variables_to_fetch=task.variables_to_fetch,
                                                                 request_timeout=zeebe_worker.request_timeout)
+
+
+stop_test = Event()
+
+
+@pytest.fixture
+def stop_after_test():
+    stop_test.clear()
+    yield
```

Comment:
Can't you yield `stop_test` instead of declaring it as a global variable?
Something like this:
```python
@pytest.fixture
def stop_after_test():
    stop_test = Event()
    yield stop_test
    stop_test.set()
```

Reply:
Addressed in 23ba49fa5647c9a617d0e1b8680b0be46f6bb266